app/views/admin/cms/_collection_list.html.erb

```diff
@@ -0,0 +1,41 @@
+<table>
+  <thead>
+    <tr>
+      <th>Collection ID</th>
+      <th>Title</th>
+      <th>Items</th>
+      <th>Published</th>
+      <th>Actions</th>
+      <th></th>
+    </tr>
+  </thead>
+
+  <tbody id="collection_list">
+    <% Collection.all.each do |resource| %>
+      <tr>
+        <td><%= resource.uid %></td>
+        <td><%= link_to resource.title, collection_path(resource.uid) %></td>
+        <td><%= resource.transcripts.size %></td>
+        <td>
+          <%= check_box_tag :is_published,
+              resource.is_published
+          %>
+        </td>
+        <td>
+          <%= button_to "Edit Collection",
+              edit_admin_cms_collection_path(resource.id),
```

Comment:
if the latter, then you shouldn't need to pass in a specific attribute at all, just the whole `resource`